src/Features/Core/Portable/EditAndContinue/ManagedEditAndContinueCapability.cs

```diff
@@ -0,0 +1,22 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+
+namespace Microsoft.CodeAnalysis.EditAndContinue
+{
+    /// <summary>
+    /// The capabilities that the runtime has with respect to edit and continue
+    /// </summary>
+    [Flags]
+    internal enum ManagedEditAndContinueCapability
+    {
+        None = 0,
+
+        Baseline = 1 << 0,
+        AddDefinitionToExistingType = 1 << 1,
```

Comment:
Unclear on what "Baseline" here means.

`AddDefinitionToExistingType`  may be too broad for Mono. I think the following will present different challenges to Mono:
- adding (non-virtual) static and instance methods
- adding static fields
- adding instance fields

Reply:
Baseline means capabilities that Mono 6, .NET FX and .NET 5 have in common.

Reply:
I've expanded out that into these constituent parts, hopefully that at least puts you in control @lambdageek 

Let me know what you think.